addons/l10n_es_edi_tbai/tests/test_edi_xml.py

```diff
@@ -43,6 +43,16 @@ def test_xml_tree_post(self):
             xml_expected = etree.fromstring(super().L10N_ES_TBAI_SAMPLE_XML_POST)
             self.assertXmlTreeEqual(xml_doc, xml_expected)
 
+    def test_xml_tree_post_generic_sequence(self):
+        with freeze_time(self.frozen_today):
```

Comment:
thanks for the test :smile:  ! I would add a docstring to explain why we have putted this test also